Automattic/jetpack
# Try/contact form gutenpack testing

This PR is created so that we can test the contact form block easier on different in Jetpack hosts.

It should not be merged. 

It consist of the wp-calypso master of the jetpack-blocks preset build the sdk in the `/inc/block/` directory.

That has the follow PR https://github.com/Automattic/wp-calypso/pull/27996

#### Changes proposed in this Pull Request:
THis should not be merged. 

#### Testing instructions:
Activate Jetpack and connect it. 
Add the new contact form block. Does it work as expected outlined in the call for testing. p1HpG7-5L1-p2
